hikari/channels.py

```diff
@@ -327,6 +327,21 @@ class PartialChannel(snowflakes.Unique):
     type: typing.Union[ChannelType, int] = attr.field(eq=False, hash=False, repr=True)
     """The channel's type."""
 
+    @property
+    def mention(self) -> str:
+        """Return a raw mention string for the channel.
+
+        !!! note
+            As of writing, GuildCategory, GroupDMChannel, and DMChannel channels are
+            a special case for this and mentions of them will not resolve as clickable.
```

Comment:
Doesn't this depend on the platform?

> Tbh, might as well move it to PartialChannels and add that GuildCategory and DmChannel don't render on all platforms correctly
> Android seems to work with everything
> Even renders categories
> But link just takes you to the first channel in the server

https://discord.com/channels/574921006817476608/644130410564943873/998865770970886194

Reply:
We are not going to through the different apps and check how they function. This is ok for now

Reply:
updated it

Reply:
I still prefer @Le0Developer's note better

Reply:
sure, i'll change it